### ISSUE TITLE
hashtags with umlauts not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#138]
```
Twitter is recently supporting hashtags containing umlauts.

If such a hashtag is encountered in pino the hashtag (which is used when
clicking on it) is cut right before the umlaut.

Example:
hastag: #enttäuscht
resulting link: http://twitter.com/#search?q= %23entt

(Also notice the leading space, but this is also present in working hashtags.)
```

Original issue reported on code.google.com by `bernd.st...@googlemail.com` on 19 Mar 2010 at 10:38

**Comment by GoogleCodeExporter** [comment]
```
Closed by f7f62eb1e8 revision.

I used this list: äëḧïöüẅẍÿ. Is it correct?
```

Original comment by `tro...@gmail.com` on 25 Mar 2010 at 12:38
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Could you please add the following to the list:

ÄÖÜßéèáàÉÈÁÀ

Thanks!
```

Original comment by `andreas....@googlemail.com` on 26 Mar 2010 at 8:03

**Comment by GoogleCodeExporter** [comment]
```
Ok, in the next version.
```

Original comment by `tro...@gmail.com` on 26 Mar 2010 at 9:54
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 1d582608c5 revision.
```

Original comment by `tro...@gmail.com` on 26 Mar 2010 at 10:45
- Changed state: **Fixed**
